core/src/blocktree.rs

```diff
@@ -105,6 +97,31 @@ pub struct SlotMeta {
     pub is_connected: bool,
 }
 
+#[cfg(feature = "erasure")]
+#[derive(Clone, Debug, Default, Deserialize, Serialize, Eq, PartialEq)]
+/// Erasure coding information
+pub struct ErasureMeta {
+    pub slot: u64,
+    /// each count is a tuple of bitfields representing presence/absence of each data blob and coding blob
+    pub counts: Vec<(u16, u8)>,
+}
+
+#[cfg(feature = "erasure")]
+impl ErasureMeta {
+    pub fn can_recover(&self, set: u64) -> bool {
```

Comment:
I think this is the crux of the change, can we add a unit test for this?

Reply:
will do